modules/Microsoft.ContainerRegistry/registries/deploy.bicep

```diff
@@ -215,7 +232,7 @@ resource cMKKeyVaultKey 'Microsoft.KeyVault/vaults/keys@2021-10-01' existing = i
   scope: resourceGroup(split(cMKKeyVaultResourceId, '/')[2], split(cMKKeyVaultResourceId, '/')[4])
 }
 
-resource registry 'Microsoft.ContainerRegistry/registries@2021-09-01' = {
+resource registry 'Microsoft.ContainerRegistry/registries@2022-02-01-preview' = {
```

Comment:
Can you update the API version of the children as well as they're also updated in the reference doc.

Reply:
done

Reply:
done